client/bin/entrypoint.sh

```diff
@@ -31,4 +31,8 @@ sed --follow-symlinks -i "s/KOJI_HUB_IP/${WORKSTATION_IP}/" /etc/osbs.conf
 sed --follow-symlinks -i "s/OPENSHIFT_IP/${WORKSTATION_IP}/" /etc/osbs.conf
 sed --follow-symlinks -i "s/REGISTRY_IP/${WORKSTATION_IP}/" /etc/osbs.conf
 
+# Replace koji IP
+KOJI_IP=$(getent hosts koji-hub | awk '{ print $1 }')
```

Comment:
I tried that and it doesn't seem to work. Apparently koji is not listening properly on exposed port (e.g. `172.21.0.1:8083`) and the connection just hangs.

I think its acceptable to use internal IP to connect to koji web UI, osbs-box is not planned to work on remote environment